### DISCUSSION TITLE
Compatibilities of Myria, Baseus, Zenkabeat and Jeswo's USB-C hubs for HDMI connection with monitors

### DISCUSSION THREAD

**Discussion by andrei-lazarov** [emoacht/Monitorian#520] [Compatibility]
Hello, 
I have tried 5 different USB-C hubs across multiple computers, monitors, HDMI cables and apps. None of them let me control the external monitor. Note that with each computer the direct HDMI to HDMI connection worked as expected. Only when connecting the monitors through hubs I got issues.
I feel like I am missing something, could all the hubs be incompatible?

Here are some probes:
[Acer_hub1_probe.log](https://github.com/emoacht/Monitorian/files/12772781/Acer_hub1_probe.log)
[Acer_hub2_probe.log](https://github.com/emoacht/Monitorian/files/12772782/Acer_hub2_probe.log)
[LG_hub1_probe.log](https://github.com/emoacht/Monitorian/files/12772783/LG_hub1_probe.log)
[LG_hub2_probe.log](https://github.com/emoacht/Monitorian/files/12772784/LG_hub2_probe.log)
[operation.log](https://github.com/emoacht/Monitorian/files/12772785/operation.log)

Thanks

**Comment by emoacht** [comment]
Thank you  for the logs.
4 probe.logs show that DDC/CI is not enabled and thus the hubs seem not compatible. Could you provide the names of the hubs for reference of other people?
The DDC/CI compatibility of a hub depends on each model. I found a Anker USB-C hub is compatible. See #475

**Comment by andrei-lazarov** [comment]
Unfortunately I can't find the Anker hub for sale in my country.

The hubs I tried are pretty cheap and generic, so I would expect them to have different names across the world. Here they are anyway: 
- [MYRIA MY8064](https://altex.ro/hub-usb-myria-my8064-usb-3-0-hdmi-vga-gri/cpd/HUBMY8064/)
- [Baseus Metal Gleam 6-in-1](https://www.emag.ro/docking-station-baseus-metal-gleam-conectare-pc-usb-type-c-gri-cahub-cw0g/pd/D0CFYJMBM/)
- [Zenkabeat 6-in-1](https://www.emag.ro/hub-adaptor-type-c-pro6-zenkabeat-6-1-3x-usb-1x-hdmi-1x-type-c-1x-port-internet-1000-mbps-negru-zk-49/pd/D69YC6MBM/)
- [No name 8-in-1](https://www.emag.ro/multiport-hub-adaptor-usb-c-8-in1-tip-c-3-1-to-4k-hd-rj45-sd-tf-cititor-card-pd-incarcare-rapida-universal-dzcp02017/pd/D2VCTBYBM/)
- [JESWO 5-in-1](https://www.emag.ro/hub-jeswo-usb-c-la-hdmi-5in1-1x-gigabit-ethernet-rj45-1x-hdmi-100w-pd-2-usb-2-0-gri-k404/pd/DJTX16MBM/)

I saw the Baseus one listed as known to work on Mac with the Lunar app. However I had no succes on Windows and Linux with multiple apps.

**Comment by emoacht** [comment]
Thank your providing the names of hubs.
I will transfer this issue to discussions for the future reference of other people.

**Comment by andrei-lazarov** [comment]
Hey I found a hub that works, maybe it helps someone:
- [Baseus 6-in-1 with right angle connector](https://www.emag.ro/adaptor-multiport-hub-6-in-1-baseus-usb-type-c-la-hdmi-4k-at30hz-usb-3-0-ethernet-si-usb-c-pd-space-grey-cahub-dz0g-179278/pd/D6CXJ8MBM/)
